Tools/msi/csv_to_wxs.py

```diff
@@ -44,7 +44,7 @@ def main(file_source, install_target):
     with open(file_source, 'r', newline='') as f:
         files = list(csv.reader(f))
 
-    assert len(files) == len(set(make_id(f[1]) for f in files)), "Duplicate file IDs exist"
+    assert len(files) == len({make_id(f[1]) for f in files}), "Duplicate file IDs exist"
```

Comment:
Skip this.  The original is clearer and the fix is one we usually don't touch unless we have to.